test-integ/peering_commontopo/testmain_test.go

```diff
@@ -1,25 +0,0 @@
-package peering
-
-import (
-	"flag"
-	"os"
-	"testing"
-)
-
-// By default, disallow running multiple commonTopo tests
```

Comment:
This comment is still relevant and helpful, but if you want to cut out the variable, it could be relocated somewhere.